Code/Source/svFSI/ComMod.h

```diff
@@ -333,6 +333,9 @@ class fibStrsType
     // Constant steady value
     double g = 0.0;
 
+    // Cross fiber stress parameter
+    double eta_s = 0.0;
```

Comment:
Is the user able to set this parameter in the input file?

Reply:
No, they can't do it in the input file as of now.

Reply:
Okay, we can leave that for a future pull request.